src/templates/layouts/healthCareLocalFacility/HealthServices/ServiceLocation.test.tsx

```diff
@@ -190,10 +190,36 @@ describe('ServiceLocation', () => {
     ).not.toBeInTheDocument()
   })
 
+  test('shows referral required when fieldReferralRequired is "1"', () => {
```

Comment:
Just noticed we were missing these tests before. Not technically related to any refactors, but it _did_ come up while I was changing things around.